src/Components/UI/Forms/ProductForm/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback } from "react";
+import { useFormik, FormikValues } from "formik";
+import { IProduct } from "../../../../models";
+import { InputText } from "primereact/inputtext";
+import styled from "styled-components";
+import { Button } from "primereact/button";
+import uuid from "react-uuid";
+import { QueryClient, useMutation } from "react-query";
+import { apiUrl, URL } from "../../../../shared";
+import { ProductService } from "../../../../shared/services/product-service";
+import { Link } from "react-router-dom";
+
+const Root = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 30px;
+`;
+
+const createProduct = (product: Partial<IProduct>) =>
+  ProductService.addProduct(product as IProduct);
```

Comment:
```suggestion
```

Reply:
i leave it like that was, call createProduct inside the useMutation